packages/runtime/src/controls/rich-text-v2/rich-text-v2.ts

```diff
@@ -54,7 +54,10 @@ export function RichText<T extends ExternalRichTextV2Config>(
     type: RichTextV2ControlType,
     config: {
       mode: config.mode,
-      defaultValue: config?.mode === RichTextV2Mode.Inline ? 'Edit this text' : ipsum(3),
+      defaultValue:
+        config?.defaultValue ?? config?.mode === RichTextV2Mode.Inline
+          ? 'Edit this text'
+          : ipsum(3),
```

Comment:
This won't work as you expect it to

![CleanShot 2023-10-10 at 18 41 55](https://github.com/makeswift/makeswift/assets/5677929/a23f9079-52bd-4047-9bc1-6e3e814909c2)

Reply:
How is that possible?

Based on this order of operations chart: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Operator_precedence#table

Wouldn't it resolve as 

```
'default_value' ?? 'one' === 'one' ? 'oh_no' : 'hi'
'default_value' ?? 'oh_no' 
'default_value'
```

Reply:
because `'default_value'` is never null or undefined.

Reply:
Ok https://stackoverflow.com/a/67575538/8239711

Reply:
The higher the number in the chart, the higher the precedence so the nullish coalesce would resolve _first_ but in your comment above you resolve it _last_